ttl/cache/table.go

```diff
@@ -124,6 +125,10 @@ func NewPhysicalTable(schema model.CIStr, tbl *model.TableInfo, partition model.
 	}, nil
 }
 
+func isTTLTable(tbl *model.TableInfo) bool {
+	return tbl.TTLInfo != nil
```

Comment:
It's better to check if the tbl.State is public

Reply:
Yes.